src/pages/components/SecondaryBar.qml

```diff
@@ -49,7 +49,14 @@ Item {
         // Spacer for pushing Search, Favorite, Share, Downloads to the right hand side
         Item {
             height: parent.height
-            width: parent.width - addTabButton.width - forwardButton.width - midIconWidth * 3 - downloadsButton.width
+            width: parent.width - addTabButton.width - forwardButton.width - midIconWidth * 4 - downloadsButton.width
+        }
+
+        Browser.IconButton {
```

Comment:
Checked the design and this could located between sharing and downloads icons so that we'd have similar kind of actions grouped together.